src/databricks/labs/ucx/install.py

```diff
@@ -490,9 +490,10 @@ def _job_settings(self, step_name: str, dbfs_path: str):
                 on_success=[self._my_username], on_failure=[self._my_username]
             )
         tasks = sorted([t for t in _TASKS.values() if t.workflow == step_name], key=lambda _: _.name)
+        version = self._version if not self._ws.config.is_gcp else self._version.replace("+", "-")
```

Comment:
I would generalize the code to make sure that all tags are cleaned up, not only the version.